distribution/src/main/resources/bin/plugin.bat

```diff
@@ -9,9 +9,46 @@ for %%I in ("%SCRIPT_DIR%..") do set ES_HOME=%%~dpfI
 
 TITLE Elasticsearch Plugin Manager ${project.version}
 
+SET properties=
+SET args=
+
+:loop
+SET "current=%~1"
+SHIFT
+IF "x!current!" == "x" GOTO breakloop
+
+IF "!current:~0,2%!" == "-D" (
```

Comment:
The idea is to model the same functionality as in the bash file:

https://github.com/elastic/elasticsearch/blob/master/distribution/src/main/resources/bin/plugin#L83-L90

This means that properties can be passed as either `-Dmy.prop true` or `-Dmy.prop=true`.


As for why the parsing is not done using `FOR /...` as in elasticsearch.bat, I could not get that approach to work with quoted parameters, for example `plugin "-Dmy.property=some value with spaces" install foo`.